tools/scripts/header_gen_historic.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+if [ "$1" = "" ]; then
+    echo Generate Free Software Header
+    echo Usage: $0 \"Filename
+    exit 1
+fi
+
+filename=`basename $1`
+author="`git log --follow --format="%an" -- $1 | tail -n 1`"
+echo $author
+email="`git log --follow --format="%ae" -- $1 | tail -n 1 `"
+echo $email
+date="`git log --follow --format="%an,%ae,%ad" -- $1 | tail -n 1 | cut -d"," -f 3`"
```

Comment:
```
date="`git log --follow --format="%ad" -- $1 | tail -n 1`"
```
? Why adding the author name and email and cutting it out again? Same for the following commands.

Reply:
Addressed in https://github.com/orocos-toolchain/rtt/pull/264/commits/27493e486f6a4bfe31052ca266fe59bbd5e95b08.